evennia/contrib/components/holder.py

```diff
@@ -0,0 +1,136 @@
+from evennia.contrib import components
+
+
+class ComponentProperty:
```

Comment:
Class docstring is needed. A descriptor like this is also tricky for newcomers to understand, so briefly describing just what it does and how it's used is useful to aid reading. Remember that this docstring will pop up in the API autodocs without users necessarily knowing to reference the README.
(See `AttributeProperty` docstring for an example)